From 9d7628d357b370b0a0ba1aab5ee809cd0cc8ee4d Mon Sep 17 00:00:00 2001
From: Al Bailey <al.bailey@windriver.com>
Date: Tue, 24 Oct 2017 11:56:26 -0500
Subject: [PATCH 02/57] Add MANIFEST.in to work with our build
 system for compile_catalog in spec file

Without a MANIFEST.in file, most of the non python files are not properly staged
and therefore we were manually including sql files
We were also failing to run:  python setup.py compile_catalog  -d blah
and now that step works.
---
 MANIFEST.in | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)
 create mode 100644 MANIFEST.in

diff --git a/MANIFEST.in b/MANIFEST.in
new file mode 100644
index 0000000..55335f6
--- /dev/null
+++ b/MANIFEST.in
@@ -0,0 +1,21 @@
+include AUTHORS
+include babel.cfg
+include ChangeLog
+include CONTRIBUTING.rst
+include LICENSE
+include HACKING.rst
+include README.rst
+include openstack-common.conf
+include run_tests.sh
+include setup.cfg
+include setup.py
+include tox.ini
+include etc/*
+include httpd/*
+graft bin
+graft doc
+graft heat/tests
+graft tools
+graft examples
+recursive-include heat *.json *.xml *.cfg *.pem README *.po *.mo *.sql
+global-exclude *.pyc *.sdx *.log *.db *.swp heat/tests/tmp/*
-- 
2.7.4

